Project-Euler/Project-Euler/Problems.cs

```diff
@@ -105,6 +103,46 @@ private static bool IsItPrime(long number)
             }
 
             return true;
-        }  
+        }
+
+        public static void Problem4()
+        {
+            Console.WriteLine("\n\t1)Find the largest palindrome made from the product of two 3 digit numbers.");
+
+            int currentProduct;
+            int greatestProduct = 0;
+
+            // This loops iterates through every single possible combination of 3-digit numbers and multiplays them. 
+            for (int i = 999; i > 99; i--)
+            {
+                for (int j = i; j > 99; j--)
```

Comment:
It was very smart to only start at i because you have already calculated the ones above whatever i is. I'm very impressed.
You may even want to make a comment pointing this smartness out